Recruit-Training/Embedded-Recruit-Training/ReadMe.adoc

```diff
@@ -0,0 +1,140 @@
+Embedded Recruit Training 
+------------------------
+
+In this recruit training you will be receiving two “control” CAN messages, one for the blue LED and the green LED. Depending on which bits are ‘set’ you will toggle either the blue or green LEDS. You will also send back a message with current state of the LEDs at a frequency of 1hz for the blue LED and 10Hz for the green. 
+
+We will begin by discussing the directory structure for the recruit training project.
+
+*  Core: where all of the user/main code will go
+** inc: where all of the header files go 
+** src: where all of the source files go 
+* Drivers: where all the HAL driver functions go
+* Middlewares: where all of the os functions (FREERTOS and CMSIS) are
+
+If there are any functions that you think you need, but aren't given, refer to Documentation/HALFunctionReferences.adoc and Documentation/RTOSReferences.adoc
+
+
+Start by going to Core/Src/main.c and search for “TODO” this will show you where you need to make changes
+
+. Within “USER CODE BEGIN Includes” and “USER CODE END Includes”
```

Comment:
Use backticks instead of quotes for this, since it's part of the code. Propagate this to similar lines :)